site/src/pages/WorkspacePage/Workspace.tsx

```diff
@@ -87,55 +90,39 @@ export const Workspace: FC<React.PropsWithChildren<WorkspaceProps>> = ({
   handleCancel,
   handleSettings,
   handleChangeVersion,
-  handleDormantActivate: handleDormantActivate,
+  handleDormantActivate,
   workspace,
   isUpdating,
   isRestarting,
   resources,
   builds,
   canUpdateWorkspace,
   updateMessage,
-  canRetryDebugMode,
   canChangeVersions,
   workspaceErrors,
   hideSSHButton,
   hideVSCodeDesktopButton,
   buildInfo,
   sshPrefix,
   template,
+  canRetryDebugMode,
   handleBuildRetry,
+  handleBuildRetryDebug,
   buildLogs,
   onLoadMoreBuilds,
   isLoadingMoreBuilds,
   hasMoreBuilds,
   canAutostart,
 }) => {
   const navigate = useNavigate();
-  const serverVersion = buildInfo?.version || "";
   const { saveLocal, getLocal } = useLocalStorage();
 
-  const buildError = Boolean(workspaceErrors[WorkspaceErrors.BUILD_ERROR]) && (
```

Comment:
These two booleans were moved into the JSX, to move them closer to where they're actually used